src/main/java/com/epam/test_generator/config/security/WebSecurityConfig.java

```diff
@@ -108,8 +107,8 @@ public CorsConfigurationSource corsConfigurationSource() {
     @Override
     public void configure(WebSecurity web) throws Exception {
         web.ignoring().antMatchers("/login", "/registration", "/v2/api-docs",
-                "/configuration/ui", "/swagger-resources/**", "/configuration/**", "/swagger-ui.html",
-                "/webjars/**","/passwordForgot","/passwordReset","/confirmAccount");
+            "/configuration/ui", "/swagger-resources/**", "/configuration/**", "/swagger-ui.html",
+            "/webjars/**", "/passwordForgot", "/passwordReset", "/confirmAccount","/static/**","/index.html");
```

Comment:
Format code. colons must be followed by spaces

Reply:
Fixed